src/java/io/bazel/rulesscala/jar/BUILD

```diff
@@ -0,0 +1,5 @@
+java_binary(name = "jar",
```

Comment:
Oh I see, then maybe you can use directly the SingleJar binary. Just ask if you need to get rid of the file fork and I can probably help you sort that out.